.eslintignore

```diff
@@ -0,0 +1,5 @@
+node_modelues/
```

Comment:
`node_modules` is by default ignored by eslint and the other files only generated during the CI build. So I feel that this file may not be of use